src/test/java/codeu/controller/RegisterServletTest.java

```diff
@@ -63,14 +59,7 @@ public void testDoPost_NewUser() throws IOException, ServletException {
 
     registerServlet.doPost(mockRequest, mockResponse);
 
-    ArgumentCaptor<User> userArgumentCaptor = ArgumentCaptor.forClass(User.class);
```

Comment:
I think you want to keep these checks.  These are verifying that that the user gets added correctly

Reply:
Since there is no User being created in the RegisterServlet anymore, wouldn't userArgumentCaptor be null since it's specifically looking for User?

Reply:
Oh, shoot, yeah, you're totally right.  You can removed the commented out stuff.  It looks good.

Reply:
Awesome, thank you!